cpp/modmesh/serialization/SerializableItem.hpp

```diff
@@ -170,6 +170,29 @@ std::string JsonHelper::to_json_string(const T & value)
         oss << "]";
         return oss.str();
     }
+    else if constexpr (is_specialization_of_v<std::unordered_map, T>)
+    {
+        static_assert(std::is_same_v<typename T::key_type, std::string>, "Only support std::unordered_map<std::string, ...>.");
```

Comment:
the key of the unordered_map should only be a string.

Reply:
I think something convertible to a string should also work, but that may complicate the code.  Type-check alone looks good right now.